.gitlab-ci.yml

```diff
@@ -349,16 +344,22 @@ build:quick:
     when: always
 
 windows64:
-  extends: .windows-template
+  extends: .platform-template
   variables:
     ARCH: "64"
+  script:
+    - call dev/ci/platform-windows.bat
+  tags:
+    - windows-inria
```

Comment:
Why's this not in the template?

Reply:
The template is for testing the platform. It is now only run on windows, but we should also run a quick job on osx and linux.

If we move to GH actions then this section goes away anyway... if not, I'd like to keep things as they are (unless one can do multiple inheritance)